explanation/security/security_suggestions.md

```diff
@@ -0,0 +1,177 @@
+   
+   * [Livepatch](https://ubuntu.com/security/livepatch) applies kernel patches
+     for high and critical severity vulnerabilities while the system is running.
+     This avoids the need for an immediate reboot.
+
+   Most security patches can be fetched and applied automatically through the
+   `unattended-upgrade` package. For more details on using and monitoring
+   Ubuntu Pro via the command line, refer to the
+   [official documentation](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/).
+
+### Access Control
+
+1. **Use and enforce** the
+   [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege):
+
+   * This means creating non-root user accounts with as few privileges as possible.
+   * Not using `sudo` (root access) except for administration tasks.
+   * For more details on basic access control, see our {ref}`guide on user management <user-management>`.
+
+### Network security
+
+1. **Use a firewall**. In Ubuntu, the uncomplicated firewall (`ufw`) tool is
+   used to configure firewalls. `ufw` is a wrapper around the `iptables` utility
+   (which experienced system admins may prefer to use directly). To get started
+   with `ufw`, check out our {ref}`firewalls` guide.
+
+1. **Use the Secure Shell (SSH)** protocol to secure remote access. In Ubuntu,
+   this is managed through OpenSSH. For details on setting up OpenSSH, refer to
+   our {ref}`guide to OpenSSH <openssh-server>`. 
+
+### Physical security
+
+There are also steps you can take to protect the physical security of your
+system. These how-to guides will help you set up these additional precautions:
+
+* {ref}`Smart card authentication <smart-card-authentication>`.
+* {ref}`Smart card authentication with SSH <smart-card-authentication-with-ssh>`.
+* {ref}`Console security <console-security>`.
+
+
+(advanced-security)=
+## Suggestions for complex setups
+
+The following section will help direct you to the security-related packages for
+which we provide documentation. For more discussion about advanced security
+considerations, refer to the [Ubuntu Security](https://ubuntu.com/security)
+documentation. 
+
+### Advanced Access Control
+
+1. **Lightweight Directory Access Protocol (LDAP)** is the usual way to gate
+   access control for larger or more complex setups. In Ubuntu, this is
+   implemented through OpenLDAP. Refer to our
+   {ref}`introduction to OpenLDAP <introduction-to-openldap>`
+   for more details, or see our section
+   {ref}`on how to set up OpenLDAP <how-to-openldap>`.
+1. **Kerberos** is a network authentication protocol that provides identity
+   verification for distributed environments, commonly used in enterprise
+   systems. Learn more in our
+   {ref}`introduction to Kerberos <introduction-to-kerberos>`, or see our
+   section on how to {ref}`set up and use Kerberos <how-to-kerberos>`.
+1. **System Security Services Daemon (SSSD)** is a collection of daemons that
```

Comment:
To be fair to all options, should we include integration into an active directory in this list?